javascript/duplicating-an-array.md

```diff
@@ -0,0 +1,18 @@
+In JavaScript it is not possible to duplicate an array simply by defining a new variable and setting its value equal to the array.
```

Comment:
Good, I think its reads better like this

'It's not possible to duplicate and array, by simply defining a new variable and setting the value equal to the array in JavaScript.'